docs/Processes.md

```diff
@@ -1064,6 +1064,59 @@ The responsibility of an IDB is therefore:
 2.  Can be triggered for a reload. This is usually done by the WDB process
     periodically.
 
+### Considerations
+
+The IDB is connected by design and operates in a space between the RDB, WDB and HDB.
+As such, the following are some considerations that must be made when creating an IDB:
+
+1.  The IDB requires both a WDB and HDB to be operational.
+
+2.  On startup, the IDB will automatically attempt to connect itself to a WDB
+    to facilitate its initialisations steps. These are:
+    - Attempt to connect to the WBD to retrieve required information:
+        - WBD location.
+        - HDB location.
+        - Current partition.
+        - Current writedown mode.
+    - Register itself with the WDB process.
+    - Loads the WDB and the HDB's sym file.
+
+3.  The IDB will have data overlapping with the RDB, so gateways and users should
+    be mindful when issuing queries.
+
+4.  The IDB is designed to fail fast so it doesn't affect surrounding components.
+    It will self-terminate as soon as an error is encountered, such as when
+    attempting to load a corrupted DB.
+
+### Querying an IDB
+
+The IDB can be queried like so:
```

Comment:
The IDB can be queried just like any other HDB. If writedown mode partbyenum is used it has a useful "maptoint" function which can be used.